unison-cli/src/Unison/Cli/Pretty.hs

```diff
@@ -371,3 +398,124 @@ prettyLabeledDependencies ppe lds =
     ld = \case
       LD.TermReferent r -> prettyHashQualified (PPE.termNameOrHashOnly ppe r)
       LD.TypeReference r -> "type " <> prettyHashQualified (PPE.typeNameOrHashOnly ppe r)
+
+prettyUnisonFile :: forall v a. (Var v, Ord a) => PPED.PrettyPrintEnvDecl -> UF.UnisonFile v a -> P.Pretty P.ColorText
```

Comment:
Moved some things here from OutputMessages since we now use them as part of commands themselves